hermes-mock/build.gradle

```diff
@@ -0,0 +1,15 @@
+plugins {
+    id 'groovy'
+    id 'java'
+}
+
+dependencies {
+    compile group: 'junit', name: 'junit', version: '4.11'
+    compile group: 'com.github.tomakehurst', name: 'wiremock', version: versions.wiremock
+    compile group: 'com.jayway.awaitility', name: 'awaitility', version: '1.6.1'
+
+    testCompile project(':hermes-test-helper')
+    testCompile group: 'org.spockframework', name: 'spock-core', version: versions.spock
+    testCompile group: 'junit', name: 'junit', version: '4.11'
```

Comment:
Do you need junit package for compilation during test phase when it is already defined in usual compilation ?